src/Behat/Behat/EventDispatcher/Cli/StopOnFailureController.php

```diff
@@ -54,10 +48,10 @@ final class StopOnFailureController implements Controller
      * Initializes controller.
      *
      * @param EventDispatcherInterface $eventDispatcher
+     * @deprecated $eventDispatcher events are now dispatched in the StopOnFailureHandler
```

Comment:
Doesn't this mark the whole constructor method as deprecated?

Reply:
this is indeed how this tag will be interpreted.

Reply:
OK, updated